actionpack/lib/action_dispatch/middleware/session/abstract_store.rb

```diff
@@ -22,6 +22,7 @@ def initialize
     module Compatibility
       def initialize(app, options = {})
         options[:key] ||= "_session_id"
+        options[:lazy] = true unless options.key?(:lazy)
```

Comment:
Not a huge fan of the name of this key ... potential alternatives: `preload`, `eager_load`, `lazy_load`